addons/binding/org.openhab.binding.netatmo/ESH-INF/config/config.xml

```diff
@@ -44,20 +40,54 @@
 		<parameter name="readHealthyHomeCoach" type="boolean" required="true">
 			<label>Access Healthy Home Coach</label>
 			<description>Read healthy home coach's data.</description>
-			<default>true</default>
+			<default>false</default>
 		</parameter>
 
 		<parameter name="readThermostat" type="boolean" required="true">
 			<label>Access Thermostat</label>
 			<description>Read and Write thermostat's data.</description>
-			<default>true</default>
+			<default>false</default>
 		</parameter>
 
+		<parameter name="readWelcome" type="boolean" required="true">
+			<label>Access Welcome camera</label>
+			<description>Read and Access Welcome camera's data.</description>
+			<default>false</default>
+		</parameter>
+
+		<parameter name="webHookUrl" type="text" required="false">
+			<label>Webhook address</label>
+			<description>Protocol, public IP and port to access OH2 server from Internet.</description>
+            <advanced>true</advanced>
+		</parameter>
+		
+        <parameter name="reconnectInterval" type="integer" required="true">
```

Comment:
Please add fixing the tabs vs spaces in this document to the after-merge task list.